php/fpm/pools_config.sls

```diff
@@ -32,7 +34,7 @@
   file.managed:
     {{ sls_block(config.get('opts', {})) }}
     - name: {{ fpath }}
-    - source: salt://php/files/php.ini
+    - source: {{ files_switch(['php.ini'], 'php_fpm_pool_conf') }}
```

Comment:
```suggestion
    - source: {{ files_switch(['php.ini'],
                              'php_fpm_pool_conf',
              ) }}
```

* Maintain layout consistency when using the `files_switch` macro.

Reply:
Done